content/apps/custom-domains.md

```diff
@@ -145,6 +111,46 @@ Rather than create the `parent.tld` domain within CF we can create the full `app
 
 [API](http://apidocs.cloudfoundry.org/206/routes/associate_app_with_the_route.html)
 
----
 
-See also: [Official Cloud Foundry Documentation](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html).
+## Managed Service Method
```

Comment:
OK! Added a note under the heading (rather than to the heading), so that we have a relatively stable anchor link for this heading.